indico/MaKaC/webinterface/pages/static.py

```diff
@@ -48,6 +48,9 @@ def _getMenu(self):
 
 class WPTPLStaticConferenceDisplay(WPStaticEventBase, WPTPLConferenceDisplay):
 
+    def getJSFiles(self):
```

Comment:
I think it would be much more appropriate to remove `getJSFiles` from `WPStaticEventBase`